src/index.html

```diff
@@ -7,8 +7,42 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+          class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list"><a href="http" class="nav__link is-active">apple</a></li>
+          <li class="list"><a href="http" class="nav__link">samsung</a></li>
+          <li class="list"><a href="http" class="nav__link">смартфоны и телефоны</a></li>
+          <li class="list">
+            <a
+              href="http"
+              class="nav__link"
+              data-qa="hover"
+            >ноутбуки и компьютеры
+            </a>
+          </li>
```

Comment:
```suggestion
          <li class="list">
            <a href="http" class="nav__link">
              смартфоны и телефоны
            </a>
          </li>
          <li class="list">
            <a
              href="http"
              class="nav__link"
              data-qa="hover"
            >
              ноутбуки и компьютеры
            </a>
          </li>
```
make the list markup consistent 😉